packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -62,8 +63,13 @@ export class TextFieldBase extends BaseComponent<ITextFieldProps, ITextFieldStat
   private _descriptionId: string;
   private _delayedValidate: (value: string | undefined) => void;
   private _isMounted: boolean;
+  /**
+   * Whether the component is controlled. Undefined means inconclusive (neither value nor
+   * defaultValue has been specified in props so far). The component may NOT change from controlled
+   * to uncontrolled or vice versa during its lifetime.
```

Comment:
The PR does actually allow it to switch though, right? It perhaps 'should not change' but for now it can (with warnings.) Do you think this will be a permanent unchanging value in the future? Probably worth clarifying this comment either way.

Reply:
The comment doesn't match the current implementation. I'll update it.